snippets/auto_gen/hubl_tags.json

```diff
@@ -79,6 +79,41 @@
         "description": "The cycle tag can be used within a for loop to cycle through a series of string values and print them with each iteration\nParameters:\n- string_to_print(String) A comma separated list of strings to print with each interation. The list will repeat if there are more iterations than string parameter values.\n",
         "prefix": "~cycle"
     },
+    "dnd_area": {
+        "body": [
+            "{% dnd_area \"my_dnd_area\"\n\tname=${name}, \n\tlabel=${label}, \n\tclass=${class}\n%}"
```

Comment:
Some history on non-self closing HubL here https://github.com/HubSpot/HubL-Language-Extension/issues/34, which is why these do not have end tags

Reply:
Working on porting the generation script to JS and will handle closing tags from there